examples/misc/lib/effective_dart/usage_bad.dart

```diff
@@ -152,6 +139,34 @@ void miscDeclAnalyzedButNotTested() {
 
 //----------------------------------------------------------------------------
 
+class Player {
+  bool get isActive => false;
+}
+
+class Team {
+  Future<List<Player>> get roster => null;
+  Future<Team> downloadTeam(String name) => null;
+
```

Comment:
How about replacing L146-148 by the following (which can help us ensure that we are using the same API in both places):

```dart
class BadTeam extends Team {
```

(Unfortunately we have to keep the local `log` field otherwise it clashes with the `log` from `math.dart`.)